src/Pool/Pool.router.ts

```diff
@@ -78,11 +92,23 @@ export class PoolRouter extends Router {
   }
 
   async getPool(req: AuthRequest) {
-    const id = server.extractFromReq(req, 'id')
-    return Pool.findOne({ id })
+    const pool_id = server.extractFromReq(req, 'id')
+    const user_id = (req.auth && req.auth.sub) || null
```

Comment:
I think we're doing this: `requestParameters.get('user_id', null)` on `getPools`. Maybe we can do the same here?

Also, as a TODO (not now), we might want to have a PermissiveAuthRequest or something of the sort for these casese

Reply:
This expression is for determining if the current user (if there is) gives like to the pool. You don't want to load a pool with the like of an arbitrary user